irony-completion.el

```diff
@@ -231,7 +231,12 @@ that can be validly accessed are deemed not-accessible."
            irony-completion-availability-filter))
    candidates))
 
-(defun irony-completion-candidates ()
+(defun irony-completion-make-prefix (prefix)
```

Comment:
I'm definitely not fond of this special case.
I would prefer to vendor `combine-and-quote-string` and make it handle the empty case.
I'm actually wondering if this should be reported to Emacs developers.